### ISSUE TITLE
Add dedicated RetroPad fast-forward button

### ISSUE THREAD

**Issue by schmurtzm** [libretro/mgba#273]
Hi, it could be useful to be able to map a key on Fast-Forward directly in core controls (without requiring to press a hotkey).
It's a good way to make the L2 and R2 keys on the gamepads and on the handhelds profitable !

[Gambatte](https://github.com/libretro/gambatte-libretro/commit/1faa9f2bcbdfbd7f4504ef43595407d56aa8fb5e), [gpSP](https://github.com/libretro/gpsp/commit/134aba2b37ca7fec28b6fcd1cd6fca0dd1abc9ab) or vice can be a good inspiration to implement that.

I've done it here (forgive the quality of the implementation, I'm not an expert at all in coding that's why I'm not making a PR 😅 ) : 
https://github.com/libretro/mgba/commit/99387c04ae2879692ee9ff199dc68b6c162f0a8a

Note that a similar implementation for rewind could be useful too 😉

**Comment by endrift** [comment]
Contrary to current "design", this should be a frontend feature. Besides, those buttons are currently bound to turbo for absolutely no reason other than someone felt like it. Again, that should be a frontend feature.

**Comment by schmurtzm** [comment]
I agree. May be there is a reason that explain that many other cores made it like this ? 🤷‍♂️
In all case that is really handy to have this feature, do you want that I create a feature request on Retroarch repo ?
